frontend/src/views/web/challenge/leaderboard.html

```diff
@@ -46,6 +49,21 @@
                                 </md-select>
                             </span>
                         </div>
+                        <div class="col xs12 s6" ng-if="challenge.isChallengeHost">
+                            <span ng-if="challenge.phaseSplitVisibility && challenge.phaseSplitId!==undefined" ng-init="challenge.isPhaseSplitVisible()">
+                                <a class="pointer" ng-click="challenge.togglePhaseSplitVisibility($event)">
+                                    <strong class="text-light-black">Public</strong>
+
+                                    <i class="fa fa-eye green-text" aria-hidden="true"></i>
+                                </a>
+                            </span>
+                            <span ng-if="!challenge.phaseSplitVisibility && challenge.phaseSplitId!==undefined" ng-init="challenge.isPhaseSplitVisible()">
+                                <a class="pointer" ng-click="challenge.togglePhaseSplitVisibility($event)">
+                                    <strong class="text-light-black">Private</strong>
+                                    <i class="fa fa-eye red-text" aria-hidden="true"></i>
+                                </a>
+                            </span>
```

Comment:
Use Single template code only. These two spans can be merged into one single span and let angular variables control what has to be displayed(ex: Private/Public, red-eye/green-eye)

Reply:
done